package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "WeatherApp",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "build": "webpack"
+  },
```

Comment:
**REQUIRED**
- Update the build script to: ` "build": "webpack --mode production"`
- Add the dev script: "dev": "webpack --mode development"

**OPTIONAL**
`"start": "webpack-dev-server --mode development --open"`